R/router.R

```diff
@@ -131,7 +137,14 @@ create_router_callback <- function(root, routes) {
         parsed <- parse_url_path(cleaned_url)
         parsed$path <- ifelse(parsed$path == "", root, parsed$path)
 
-        if (!valid_path(routes, parsed$path)) {
+        path_validation <- if (is.null(routes)) {
```

Comment:
Suggestion: rename `path_validation` to `is_path_valid` - as I understand it's a check if the path used in the url matches any of the routes defined in the router, right?